charmtools/build/tactics.py

```diff
@@ -253,6 +235,40 @@ def __str__(self):
         return "Bind Interface {}".format(self.interface.name)
 
 
+class StorageBind(Tactic):
```

Comment:
Should this share a baseclass with interface bind and share code. replicating things like sign() happen too much already in the code base. I won't insist, but if you agree that is a good idea we can make an issue.